src/main/java/lotto/domain/Lotteries.java

```diff
@@ -24,6 +24,12 @@ public Lotteries(Lotto... lotto) {
 		}
 	}
 
+	public void add(Lotteries newLotteries) {
+		newLotteries.lotteries.stream().forEach(lotto -> {
+			this.lotteries.add(lotto);
+		});
```

Comment:
받은 객체가 같은 타입이므로 간단하게 줄여볼 수 있을 것 같아요 :)
```suggestion
		this.lotteries.addAll(newLotteries.lotteries);
```